### PR TITLE
Includes my program for the Turtle exercise

### PR DESCRIPTION
**Pull request by samandalib**

### CODE REVIEW

**Comment by samandalib** [comment]
Since I was behind the class schedule, I tried to make my turtle exercise as one that can cover all the topics discussed in three weeks that I missed. It covers : Custom Turtle, Logical Turtles Exercises, Turtlehacks, and Functional Turtles Exercise (It makes me a week ahead!).

File: turtleProgram [hunk 1]
```diff
@@ -0,0 +1,12 @@
+---
+Layout: post
+author: Hesam
```
Comment (eah13) [inline turtleProgram]: This needs to be your github username

**Review by eah13 — CHANGES_REQUESTED** [review]
Check the filename of your program. There's a very specific format you'll need to follow.

File: turtleProgram (hunk 1 above)
Comment (samandalib) [reply turtleProgram]: Sorry about that! fixed it

**Review by eah13 — APPROVED** [review]
Nicely done.

**Pull request merged**